#! /bin/sh /usr/share/dpatch/dpatch-run
## 30_actionsfile_fullname.dpatch by Roland Rosenfeld <roland@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The "actionsfile" syntax in the configuration file is consistent
## DP: with the rest of the configuration options and requires the
## DP: whole file name.

@DPATCH@
diff -urNad privoxy~/config privoxy/config
--- privoxy~/config
+++ privoxy/config
@@ -334,10 +334,10 @@
 #      privacy considerations, etc. There is no point in using Privoxy
 #      without at least one actions file.
 #
-actionsfile standard  # Internal purpose, recommended
-actionsfile global    # Global default setting for all sites
-actionsfile default   # Main actions file
-actionsfile user      # User customizations
+actionsfile standard.action  # Internal purpose, recommended
+actionsfile global.action    # Global default setting for all sites
+actionsfile default.action   # Main actions file
+actionsfile user.action      # User customizations
 
 #
 #  2.4. filterfile
